docs/getting-started/flatbuffers.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Read, write, and visualize MCAP files containing Flatbuffer data.
+---
+
+# Flatbuffers
+
+## Read and write MCAP
+
+If you're starting from scratch, you can write code that allows you to **write your Flatbuffer data to MCAP files** and subsequently **read your Flatbuffer data from your MCAP files**.
```

Comment:
Audit all uses of "FlatBuffers" and use appropriate capitalization and pluralization.

Reply:
good call 😅